microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -19,6 +19,18 @@ On Windows 11, widgets appear in the widgets board which you open from the left
 In Windows 11, Progressive Web Apps (PWAs) can define widgets, update them, and handle user interactions within them.
 
 
+<!-- ------------------------------ -->
+#### Requires developing a PWA as a widget
+
+For a PWA-driven widget, you build the widget experience by using the web app manifest, Adaptive Card templates, and JavaScript, rather than putting an existing PWA into a widget.  A PWA-driven widget is a separate experience, distinct from PWAs in the sidebar, which takes the entire PWA and puts it in the sidebar.
```

Comment:
It seems like these docs typically don't use "you" that often. Should this be rephrased to remove the "you"s?

Reply:
General notes about "you":
It's good to only use "you" or "we" when needed, and keep focus on the tech where possible.

29 hits on "you" in the article's source file:
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/5093ae9f-1f63-4bad-bc1a-80dd7d2878f7)

Style guide:
https://learn.microsoft.com/en-us/style-guide/grammar/person#in-general-use-second-person-pronouns-you-your - "In general, use second-person pronouns (you, your)"

Specific notes about "you" in this section:
tbd